realtime_trains/transform.py

```diff
@@ -0,0 +1,95 @@
+'''Transforms and cleans data from RealTime Trains API'''
+
+import logging
+from dotenv import load_dotenv
+from extract import get_realtime_trains_data, save_data_to_file
+
+LOCATION_REMOVE_KEYS = [
+    'tiploc',
+    'country',
+    'system'
+]
+SERVICE_REMOVE_KEYS = [
+    "trainIdentity",
+    "runningIdentity",
+    "isPassenger"
+]
+SERVICE_LOC_DETAILS_REMOVE_KEYS = [
+    'realtimeActivated',
+    'origin',
+    'destination',
+    'isCall',
+    'isPublicCall',
+    'realtimeArrivalNextDay',
+    'realtimeDepartureNextDay',
+    'platform',
+    'platformConfirmed',
+    'platformChanged'
+]
+SERVICE_CRITERIA = {
+    "serviceType": "train"
+}
```

Comment:
Can you explain your reasoning behind removing these keys, as opposed to creating a new dict with the keys you want?

Reply:
Occasionally, when the service is canceled or is any other status besides CALL, additional keys are added and therefore if I used an exclusive approach with the keys in the dictionary, there's a chance that this additional information will not be included.